src/systempolicy/systemPolicy.go

```diff
@@ -727,6 +728,7 @@ func hashInt(s string) uint32 {
 func mergeSysPolicies(pols []types.KnoxSystemPolicy) []types.KnoxSystemPolicy {
 	var results []types.KnoxSystemPolicy
 	for _, pol := range pols {
+		pol.Metadata["name"] = "autopol-system-" + strconv.FormatUint(uint64(hashInt(pol.Metadata["labels"]+pol.Metadata["namespace"]+pol.Metadata["containername"])), 10)
```

Comment:
We cannot keep containername for hash creation, since a single pod might have multiple container and it will lead to different files for the same pod.